components/cdc/src/initializer.rs

```diff
@@ -241,15 +222,34 @@ impl<E: KvEngine> Initializer<E> {
             Scanner::TxnKvScanner(txnkv_scanner)
         } else {
             let mut iter_opt = IterOptions::default();
-            iter_opt.set_lower_bound(&[RAW_KEY_PREFIX], 1);
-            iter_opt.set_upper_bound(&[RAW_KEY_PREFIX + 1], 1);
+            iter_opt.set_lower_bound(&[RAW_KEY_PREFIX], DATA_KEY_PREFIX_LEN);
+            iter_opt.set_upper_bound(&[RAW_KEY_PREFIX + 1], DATA_KEY_PREFIX_LEN);
             let mut iter = RawMvccSnapshot::from_snapshot(snap).iter(iter_opt).unwrap();
 
             iter.seek_to_first().unwrap();
             Scanner::RawKvScanner(iter)
         };
 
+        fail_point!("cdc_incremental_scan_start");
```

Comment:
Why move this block of codes ?

Reply:
https://github.com/tikv/tikv/pull/12387#discussion_r860462211 
I want to restore the order of code.